account_fiscal_position_partner_type/models/account_fiscal_position.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2021 Valentin Vinagre <valentin.vinagre@sygel.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class AccountFiscalPosition(models.Model):
+    _inherit = "account.fiscal.position"
+
+    fiscal_position_type = fields.Selection(
+        selection=[("b2c", "Customer"), ("b2b", "Company")],
+        string="Type",
+        default=lambda self: self.env.company.default_fiscal_position_type,
```

Comment:
is it possible to change this to a method?

default=lambda self: self._default_fiscal_position_type(),
    )

    @api.model
    def _default_fiscal_position_type(self):
        return self.env.company.default_fiscal_position_type

Reply:
Done